src/com/geekhub/hw2/parts/carsParts/CarEngine.java

```diff
@@ -0,0 +1,24 @@
+package com.geekhub.hw2.parts.carsParts;
```

Comment:
Better use lower case for packages names.  Also packages:  boatParts, carParts etc. are already in "parts" package, so you can name them just "boat", "car" etc.